README.md

```diff
@@ -15,43 +15,78 @@ metabolic network, namely Flux Balance Analysis and Flux Variability Analysis.
 [![Tutorial In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/GeomScale/dingo/blob/develop/tutorials/dingo_tutorial.ipynb)
 [![Chat](https://badges.gitter.im/geomscale.png)](https://gitter.im/GeomScale/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link)
 
+
 ## Installation
 
+**Note:** Python version should be 3.8.x. You can check this by running the following command in your terminal:
```

Comment:
I have tried multiple combinations of Python and poetry versions and followed the same installation instructions, below is the output of the `poetry install` command 

Python 3.10.13 and Poetry (version 1.8.2) [If installed using `pip install poetry`]
![image](https://github.com/GeomScale/dingo/assets/93584728/34db4607-e058-4321-a0ff-f091a914b451)

Python 3.8.18 and Poetry (version 1.8.2) [If installed using `pip install poetry`]
![image](https://github.com/GeomScale/dingo/assets/93584728/cc73e473-80bb-42fa-8a22-ff69756d06a6)

Python 3.10.13 and Poetry (version 1.3.2)
![image](https://github.com/GeomScale/dingo/assets/93584728/3d6e7a66-c9ed-47d7-95ec-e95bee73640b)

Python 3.8.18 and Poetry (version 1.3.2) 
![image](https://github.com/GeomScale/dingo/assets/93584728/73147600-aa2d-4d89-9406-500c32433b3e)

Reply:
great, could you please list all the possible choices and the possible solutions/workarounds in the README. 
e.g. Python 3.8.18, Poetry 1.3.2 -- OK
Python 3.10.13, Poetry  1.3.2 -- fail

BTW why the second case above fails? Is there a workaround?

Reply:
I tried with multiple other combinations of Python and Poetry versions, but only  Python 3.8.18 with Poetry 1.3.2 seems to work consistently, while other combinations like Python 3.10.13 with Poetry 1.3.2 fails due to dependency compatibility issues. This is likely because some libraries haven't been updated for other python versions such as Python 3.10 yet. I tried workarounds like specifying broader version ranges (e.g., numpy==^1.20.1) in the pyproject.toml file, and this doesn't seems to help.